### ISSUE TITLE
Running error in raspberry pi 3

### ISSUE THREAD

**Issue by neilwang0913** [gprMax/gprMax#309]
Hello: 

I am fellow the instruction for my raspberry pi 3 and everything goes well until running : 
%python setup.py install

It gives me the following error:
> KeyError: "There is no item named 'nbconvert/tests/exporter_entrypoint/eptest-0.1.dist-info/WHEEL' in the archive"

Any suggestion to run the gprmax on Rpi?

**Comment by craig-warren** [comment]
@neilwang0913 I applaud you for trying to get gprMax running on a Raspberry Pi 3....however, it is not something we have, or are currently able, to test on, so can't help you.

**Comment by neilwang0913** [comment]
Thank you. It will be great to run the edge device as they are cheaper and can be assembled easily as a supercomputer in some case

**Comment by swenkel** [comment]
Using conda environments on the RP 3 is a bit tricky and the installer might not always work. However, gprMax works out of the box on a RP 3 (tested with the 64 bit version of Raspberry OS) the when all [dependencies](https://github.com/gprMax/gprMax/blob/master/conda_env.yml) (except pycuda) are installed via the apt package manager.

You need to run  `python3 setup.py install` either as root (if you trust gprMax) or tell it to build/install it for the local user.

Running `$ python3 -m gprMax user_models/cylinder_Ascan_2D.in` outputs:

```
=== Electromagnetic modelling software based on the Finite-Difference Time-Domain (FDTD) method =============================================================================================

    www.gprmax.com   __  __
     __ _ _ __  _ __|  \/  | __ ___  __
    / _` | '_ \| '__| |\/| |/ _` \ \/ /
   | (_| | |_) | |  | |  | | (_| |>  <
    \__, | .__/|_|  |_|  |_|\__,_/_/\_\
    |___/|_|
                     v3.1.6 (Big Smoke)

 Copyright (C) 2015-2022: The University of Edinburgh
 Authors: Craig Warren and Antonis Giannopoulos

 gprMax is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the
  License, or (at your option) any later version.
 gprMax is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  General Public License for more details.
 You should have received a copy of the GNU General Public License along with gprMax.  If not, see www.gnu.org/licenses.

Host: localhost | unknown unknown | 1 x unknown (4 cores) | 870MiB RAM | Linux-5.15.32-v8+-aarch64-with-glibc2.31

--- Model 1/1, input file: user_models/cylinder_Ascan_2D.in ---------------------------------------------------------------------------------------------------------------------------------

Constants/variables used/available for Python scripting: {c: 299792458.0, current_model_run: 1, e0: 8.8541878128e-12, inputfile: /home/pi/geophysics/gprMax/user_models/cylinder_Ascan_2D.in, m0: 1.25663706212e-06, number_model_runs: 1, z0: 376.73031366686166}

Model title: A-scan from a metal cylinder buried in a dielectric half-space
Number of CPU (OpenMP) threads: 4
Spatial discretisation: 0.002 x 0.002 x 0.002m
Domain size: 0.24 x 0.21 x 0.002m (120 x 105 x 1 = 12600 cells)
Mode: 2D TMz
Time step (at CFL limit): 4.71731e-12 secs
Time window: 3e-09 secs (637 iterations)

Waveform my_ricker of type ricker with maximum amplitude scaling 1, frequency 1.5e+09Hz created.
Hertzian dipole with polarity z at 0.1m, 0.17m, 0m, using waveform my_ricker created.
Receiver at 0.14m, 0.17m, 0m with output component(s) Ex, Ey, Ez, Hx, Hy, Hz created.
Material half_space with eps_r=6, sigma=0 S/m; mu_r=1, sigma*=0 Ohm/m created.
Geometry view from 0m, 0m, 0m, to 0.24m, 0.21m, 0.002m, discretisation 0.002m, 0.002m, 0.002m, with filename base cylinder_half_space created.

Memory (RAM) required: ~51.5MB

Box from 0m, 0m, 0m, to 0.24m, 0.17m, 0.002m of material(s) half_space created, dielectric smoothing is on.
Cylinder with face centres 0.12m, 0.08m, 0m and 0.12m, 0.08m, 0.002m, with radius 0.01m, of material(s) pec created, dielectric smoothing is off.
Processing geometry related cmds: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00, 182.04cmds/s]

PML: formulation: HORIPML, order: 1, thickness: x0: 10, y0: 10, z0: 0, xmax: 10, ymax: 10, zmax: 0 cells
Building PML boundaries: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<00:00, 131.77it/s]

Building main grid: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00, 76.18it/s]

Materials:
    |                                             |                     |       | sigma |      | sigma*  | Dielectric
 ID | Name                                        | Type                | eps_r | [S/m] | mu_r | [Ohm/m] | smoothable
----+---------------------------------------------+---------------------+-------+-------+------+---------+------------
  0 | pec                                         | builtin             | 1     | inf   | 1    | 0       | False
  1 | free_space                                  | builtin             | 1     | 0     | 1    | 0       | True
  2 | half_space                                  |                     | 6     | 0     | 1    | 0       | True
  3 | free_space+free_space+half_space+half_space | dielectric-smoothed | 3.5   | 0     | 1    | 0       | True

Numerical dispersion analysis: estimated largest physical phase-velocity error is -0.42% in material 'half_space' whose wavelength sampled by 14 cells. Maximum significant frequency estimated as 4.32623e+09Hz

Writing geometry view file 1/1, cylinder_half_space.vti: 100%|███████████████████████████████████████████████████████████████████████████████████████| 75.6k/75.6k [00:00<00:00, 14.4Mbyte/s]

Output file: /home/pi/geophysics/gprMax/user_models/cylinder_Ascan_2D.out

Running simulation, model 1/1: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 637/637 [00:03<00:00, 190.79it/s]
Memory (RAM) used: ~58.9MB
Solving time [HH:MM:SS]: 0:00:03.341760

=== Simulation completed in [HH:MM:SS]: 0:00:03.748275 ======================================================================================================================================

```

**Comment by neilwang0913** [comment]
Yes, I think 64bit OS is the key
